src/middleware/error.middleware.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
@pichu07 Why is it done twice?

Reply:
It was to ignore the warning for unused variables that were necessary to define in function parameters otherwise it was throwing error